include/rocksdb/table_properties.h

```diff
@@ -193,6 +195,15 @@ struct TableProperties {
   // Actual SST file creation time. 0 means unknown.
   uint64_t file_creation_time = 0;
 
+  // DB identity
```

Comment:
Add one or two sentences to explain the identity and session id.

Reply:
Sure!